.circleci/config.yml

```diff
@@ -44,9 +44,32 @@ jobs:
 
       - run: npm test
 
+  docs:
```

Comment:
Might be worth using an "anchor" for some of these repeated blocks. https://learnxinyminutes.com/docs/yaml/ I would point to the yaml spec but it's like 200 pages